flink-clients/src/main/java/org/apache/flink/client/cli/AbstractCustomCommandLine.java

```diff
@@ -42,6 +47,8 @@
  */
 public abstract class AbstractCustomCommandLine implements CustomCommandLine {
 
+	private static final Logger LOG = LoggerFactory.getLogger(AbstractCustomCommandLine.class);
```

Comment:
Maybe we can make this logger `protected final Logger LOG = LoggerFactory.getLogger(getClass());` for all its sub classes to use, so that we do not need to create multiple loggers.

Reply:
Some static methods need to use the logger. So i just leave it to different implementation class.